pkg/transforms/genericResourceConfig.go

```diff
@@ -126,10 +126,19 @@ var defaultTransformConfig = map[string]ResourceConfig{
 		properties: []ExtractProperty{
 			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
 			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "phase",  JSONPath: `{.status.phase}`},
 			{Name: "indications", JSONPath: `{.status.indications}`}, // this is an array of strings - will collect array items separated by ;
 			{Name: "sourceVM", JSONPath: `{.spec.source.name}`},
 		},
 	},
+	"VirtualMachineRestore.snapshot.kubevirt.io": {
+		properties: []ExtractProperty{
+			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
+			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "lastRestoreTime", JSONPath: `{.status.restoreTime}`},
+			{Name: "targetVM", JSONPath: `{.spec.target.name}`},
```

Comment:
Would it be better to store this as a relationship?

Reply:
It should likely be a relationship as well.
However, there are a couple of places in the UI where I need to know the target/source VirtualMachine. With a relationship, I would have to trigger another search request. 
Having it in the resources search data already enables me to bypass having to perform another search query in the UI.

Reply:
It is not currently displayed in the UI. Typically, in VNC, the user will already be in context of the target/source VM when viewing a restore/snapshot resource.

If a user is viewing a restore/snapshot in context of search I think they should either be able to search for target/source VM OR find it via related resource section.